src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -485,10 +485,21 @@ async function getHermesCommandFromGradle(gradleFile: string): Promise<string> {
   }
 }
 
-export function getAndroidHermesEnabled(gradleFile: string): boolean {
-  return parseBuildGradleFile(gradleFile).then((buildGradle: any) => {
-    return Array.from(buildGradle["project.ext.react"] || []).some((line: string) => /^enableHermes\s{0,}:\s{0,}true/.test(line));
-  });
+export async function getAndroidHermesEnabled(gradleFile: string): Promise<boolean> {
+  // find hermesEnabled in gradle.properties first
```

Comment:
I guess the comment is misplaced. Let's put it right under the parsing `gradle.properties`.